content/docs/rfcs/53/README.md

```diff
@@ -0,0 +1,250 @@
+---
+slug: 53
+title: 53/WAKU2-X3DH
+name: X3DH usage for Waku payload encryption
+status: stable
+category: Standards Track
+tags: waku-application
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors:
+- Andrea Piana <andreap@status.im>
+- Pedro Pombeiro <pedro@status.im>
+- Corey Petty <corey@status.im>
+- Oskar Thorén <oskar@status.im>
+- Dean Eigenmann <dean@status.im>
+---
+
+# Abstract
+
+This document describes a method can be used to provide a secure channel between two peers, and thus provide confidentiality, integrity, authenticity and forward secrecy. 
+It is transport-agnostic and works over asynchronous networks.
+
+It builds on the [X3DH](https://signal.org/docs/specifications/x3dh/) and [Double Ratchet](https://signal.org/docs/specifications/doubleratchet/) specifications, with some adaptations to operate in a decentralized environment.
+
+# Background
+
+This document describes how nodes establish a secure channel, and how various conversational security properties are achieved.
+
+# Specification
```

Comment:
hmm. I figured we needed motivation instead of background, does this work - [241a5f4](https://github.com/vacp2p/rfc/pull/569/commits/241a5f4fd9e14b30417668df8951a61e05e7ff40)?